client/my-sites/checkout/checkout-thank-you/header.jsx

```diff
@@ -252,6 +253,16 @@ export class CheckoutThankYouHeader extends PureComponent {
 		window.location.href = selectedSite.URL;
 	};
 
+	visitSiteHostingSettings = event => {
+		event.preventDefault();
+
+		const { selectedSite } = this.props;
+
+		this.props.recordTracksEvent( 'calypso_thank_you_back_to_hosting' );
+
+		window.location.href = `/hosting-config/${ selectedSite.slug }`;
```

Comment:
non-blocking and I know this is copy pasta from the page, but there is a potential race condition here where the tracks event may not complete before the browser navigates to a new section.

In a follow up janitorial, we can update these actions to use the `NAVIGATE` redux action, that is [decorated with meta analytics](https://github.com/Automattic/wp-calypso/blob/c1d1fac0fad54945ddb8b2a119e498fb07e97ed6/client/state/analytics/actions/with-analytics.js#L17), so two side-effects occur:

 - The analytics middleware sees the meta value on the action and fires the related tracks event.

https://github.com/Automattic/wp-calypso/blob/c1d1fac0fad54945ddb8b2a119e498fb07e97ed6/client/state/analytics/middleware.js#L75

- The navigation middleware, triggers a `page` router call

https://github.com/Automattic/wp-calypso/blob/c1d1fac0fad54945ddb8b2a119e498fb07e97ed6/client/state/navigation/middleware.js#L43